README.md

```diff
@@ -2,14 +2,13 @@
 
 # Project Name
 
-> Description the project.
+> Portfolio project.
 
 
```

Comment:
- [ ] Kindly make sure to edit each section in your readme with the appropriate information to better suit your project and make it more professional. This also means that if there are sections that you don't have information for yet, then they should be removed. (Like the second author for the "Authors" section, "Run tests" section, "Deployment" section, ...etc)